public/js/user.js

```diff
@@ -82,8 +87,28 @@ apos.define('apostrophe-workflow', {
 
         setClass($menu, 'apos-workflow-editable', result.modified.length || result.unmodified.length);
         setClass($menu, 'apos-workflow-modified', !!result.modified.length);
+
+        var submittableOnly = _.filter(unsubmitted, function(id) {
+          return !_.find(result.committable, function(cid) {
+            return id === cid;
+          });
+        });
+
+
         setClass($menu, 'apos-workflow-committable', !!result.committable.length);
-        setClass($menu, 'apos-workflow-unsubmitted', !!unsubmitted.length);
+
+        if (self.options.committersSeeSubmit) {
+          setClass($menu, 'apos-workflow-unsubmitted', !!unsubmitted.length);
+        } else {
+          if (submittableOnly.length) {
+            setClass($menu, 'apos-workflow-unsubmitted', true);
+            setClass($menu, 'apos-workflow-submit-irrelevant', false);
```

Comment:
I'd suggest `apos-workflow-submit-hidden` or `apos-workflow-no-submit` for simplicity.

Reply:
Done.